npc/pre-re/jobs/1-1/archer.txt

```diff
@@ -117,10 +117,12 @@ payon_in02,64,71,4	script	Archer Guildsman#archer	85,{
 					close;
 				}
 			}
-			mes "[Archer Guildsman]";
-			mes "Well, alright.";
-			mes "See you next time.";
-			close;
+			else {
```

Comment:
With this "else", the above script continues (select case 1) and the player has access to dialog to which he didn't have access.

btw I don't see what was the problem here :
```
	mes "[Archer Guildsman]";
	mes "Alright, let me check.";
	next;
	mes "[Archer Guildsman]";
	mes "Well, alright.";
	mes "See you next time.";
	close;
```
seems "fine"

Reply:
According to the official script, after selecting "Yes, I am.", it should continue like this: 

```
mes "[Archer Guildsman]";
mes "Alright, let me check.";
next;

mes "[Archer Guildsman]";
mes "Are you..." + strcharinfo(0) + "?";
```

so yeah, the else is actually needed, and should only appear when you select "Cancel" in the Apply/Cancel menu